.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "Bicep Authoring devcontainer",
+	"dockerFile": "Dockerfile",
+	"forwardPorts": [ 7071 ],
+
+	// Set *default* container specific settings.json values on container create.
+	"settings": {
+		"terminal.integrated.shell.linux": "/usr/bin/pwsh"
+	},
+
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"/vscode-bicep.vsix",
+		"ms-vscode.powershell"
+	],
```

Comment:
I think it would be also worth installing the "msazurermtools.azurerm-vscode-tools" extension here.

Reply:
Agree, added.